Sources/Commands/SwiftPackageCollectionsTool.swift

```diff
@@ -212,9 +212,13 @@ public struct SwiftPackageCollectionsTool: ParsableCommand {
             guard let version = version else {
                 return nil
             }
+            // FIXME: do we want to display all manifests?
+            guard let manifest = version.manifests.default else {
```

Comment:
Here we display only a single manifest (the one with the latest tools version). Don't know if we want to show more?